### ISSUE TITLE
Narrator is not announcing state (expanded/collapsed) after activating "Choose a panel Size" Dropdown

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [microsoft/fluentui#12820]
User Experience:
Users who rely on screen readers are impacted as narrator is not announcing state (expanded/collapsed) after activating "New Extractor Name" combo edit.

Test Environment:
OS: Windows 10
Version: 2004(OS Build 19577.1000)
Browser: Edge - Canary (82.0.458.0)
Screen Reader: Narrator
URL 1: https://developer.microsoft.com/en-us/fluentui#/controls/web/panel
URL 2: https://fabricweb.z5.web.core.windows.net/oufr/7.107.3/#/examples/panel

Repro Steps:
Step 1: Open the above 2 URL's in Edge - Canary browser.
Step 2: Login with valid credentials.
Step 3: Navigate to "Choose a panel Size" combo box
Step 4: Verify behavior of screen reader after expanding/collapsing "Choose a panel Size" combo box

Actual:
Narrator is not announcing state (expanded/collapsed) after activating "Choose a panel Size" combo box

Expected:
Narrator should announce state (expanded/collapsed) after activating "Choose a panel Size" combo box
![image](https://user-images.githubusercontent.com/53210862/79998483-61647f80-84d8-11ea-82f7-70a89b30705a.png)

Note: Issue repro in both URL's

**Comment by joschect** [comment]
@SyedRehmanRazvi Thanks for the submission! I'm able to reproduce this. As far as I can tell dropdown has the appropriate `aria-expanded` tags for when the dropdown is open or closed so I'm not sure why it's being read. We'll have to investigate further.

**Comment by opga11ytest** [comment]
Verified on below URL and found screen reader announce, ‘Choose a panel size small default small default combo box collapsed has pop up’ and after activating the combo box screen reader announces ’Choose a panel size small default one of nine selected’. This information is sufficient for the screen reader users.

Environment Details:
URL: Home - Fluent UI (microsoft.com)
OS Version: 21H2 (OS Build 22000.100)
Edge Dev Version: Version 93.0.961.10 (Official build) dev (64-bit)
Fabric Version: 8.26.2
